reference/feature-toggles/available-feature-toggles.md

```diff
@@ -30,3 +30,7 @@ with a `boolean` value.
 
   If enabled, it is possible to edit DMN tables from the ui from the admin page. This feature is experimental, please
   refer to [the release notes](../../release-notes/major9/09.20.0/valtimo-frontend-libraries.md) for known issues.
+
+* **`caseSearchFields`**
```

Comment:
It's a bit too late for this, so I'm approving the PR. But I feel like we should have included demo somewhere in the fieldname.